### PR TITLE
Locale

### PR DESCRIPTION
**Pull request by LAfricain**

### CODE REVIEW

**Comment by LAfricain** [comment]
Il closed an other pull request #969 bucause this one is more complet.

**Comment by alerque** [comment]
In the future note you can extend or teak the contents of Pull Requests while they are open just by pushing more commits to the same branch.

**Comment by LAfricain** [comment]
Thank you for the advice. Yes it's better to do like you proposed.

**Comment by LAfricain** [comment]
I wanted to change also cop, but a commentary says issues happen with the font. Is this still relevant?

File: ui/languages [hunk 1]
```diff
@@ -2469,8 +2482,8 @@ gab	Gabri
 gad	Gaddang
 gae	Guarequena
 gaf	Gende
-gag	Gagauz
-#gag-Cyrl	Gagauz
+#Gagauz
+gag	Гагаузы
```
Comment (alerque) [inline ui/languages:2486]: This entry is incorrect, the reason there is a `-Cyrl` sub-locale in that the default Gagauz alphabet is a Latin based one similar to Turkish.

File: ui/languages [hunk 2]
```diff
@@ -5537,6 +5561,8 @@ oru	Ormuri
 orv	Old Russian
 orw	Oro Win
 orx	Oro
+# Orya
```
Comment (alerque) [inline ui/languages:5564]: Why have Latin comments for a handful of these entries an not others? Can we just drop the English comments, this is a locale list, the codes should identify them.

File: ui/languages [hunk 3]
```diff
@@ -6151,9 +6177,9 @@ sad	Sandawe
 sae	Sabanês
 saf	Safaliba
 sag	Sango
-sah	Yakut
-# as above, in cyrillic
-sah-Cyrl	Yakut
+# Yakut
+sah	Саха тыла
```
Comment (alerque) [inline ui/languages:6181]: This entry should not be Cyrillic, that's why there is a `-Cyrl` variant, no?

File: ui/languages (hunk 2 above)
Comment (LAfricain) [reply ui/languages:5563]: I have no opinion about this. The old entries had comments with the english name, but not the 3 letters entries. I added the English comments when it was in cyrillic.

**Pull request merged**